src/main/java/de/dentrassi/crypto/pem/AbstractPemKeyStore.java

```diff
@@ -92,7 +94,7 @@ public Entry merge(final Entry other) {
         }
     }
 
-    private Map<String, Entry> entries = Collections.emptyMap();
+    private Map<String, Entry> entries = new HashMap<String, AbstractPemKeyStore.Entry>();
```

Comment:
You can drop the explicitly declared types on `HashMap` here.  The following should work just fine

```suggestion
    private Map<String, Entry> entries = new HashMap<>();
```

Reply:
It's done.